sw/airborne/modules/rot_wing_drone/wing_rotation_controller_servo.c

```diff
@@ -0,0 +1,254 @@
+/*
+ * Copyright (C) 2022 Dennis van Wijngaarden <D.C.vanWijngaarden@tudelft.nl>
+ *
+ * This file is part of paparazzi
+ *
+ * paparazzi is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2, or (at your option)
+ * any later version.
+ *
+ * paparazzi is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with paparazzi; see the file COPYING.  If not, see
+ * <http://www.gnu.org/licenses/>.
+ */
+
+/** @file "modules/rot_wing_drone/wing_rotation_controller_v3b.c"
+ * @author Dennis van Wijngaarden <D.C.vanWijngaarden@tudelft.nl>
+ * Module to control wing rotation servo command based on prefered angle setpoint
+ */
+
+#include "modules/rot_wing_drone/wing_rotation_controller_servo.h"
+#include "modules/radio_control/radio_control.h"
+#include "firmwares/rotorcraft/guidance/guidance_h.h"
+
+#include "state.h"
+
+#include <stdlib.h>
+#include "mcu_periph/adc.h"
+
+/*
+#ifndef WING_ROTATION_SERVO_IDX
+#error "No WING_ROTATION_SERVO_IDX defined"
+#endif
+*/
+
+#if !USE_NPS
+
+#ifndef ADC_CHANNEL_WING_ROTATION_POSITION
+#define ADC_CHANNEL_WING_ROTATION_POSITION ADC_5
+#endif
+
+#ifndef ADC_CHANNEL_WING_ROTATION_POSITION_NB_SAMPLES
+#define ADC_CHANNEL_WING_ROTATION_POSITION_NB_SAMPLES 16
+#endif
+
+#endif // !USE_NPS
+
+#ifndef WING_ROTATION_POSITION_ADC_0
+#error "No WING_ROTATION_POSITION_ADC_0 defined"
+#endif
+
+#ifndef WING_ROTATION_POSITION_ADC_90
+#error "No WING_ROTATION_POSITION_ADC_90 defined"
+#endif
+
+#ifndef WING_ROTATION_FIRST_DYN
+#define WING_ROTATION_FIRST_DYN 0.001
+#endif
+
+#ifndef WING_ROTATION_SECOND_DYN
+#define WING_ROTATION_SECOND_DYN 0.003
+#endif
+
+// Parameters
+struct wing_rotation_controller wing_rotation;
+
+bool in_transition = false;
+
+#if !USE_NPS
+static struct adc_buf buf_wing_rot_pos;
+#endif
+
+#if USE_NPS
+#include "modules/actuators/actuators.h"
+#endif
+
+// Inline functions
+inline void wing_rotation_to_rad(void);
+inline void wing_rotation_update_sp(void);
+inline void wing_rotation_compute_pprz_cmd(void);
+
+void wing_rotation_init(void)
+{
+  // your init code here
+#if !USE_NPS
+  adc_buf_channel(ADC_CHANNEL_WING_ROTATION_POSITION, &buf_wing_rot_pos, ADC_CHANNEL_WING_ROTATION_POSITION_NB_SAMPLES);
+#endif
+
+  // Init wing_rotation_controller struct
+  wing_rotation.servo_pprz_cmd = 0;
```

Comment:
Just do `wing_rotation = {0};` after that just set the variables which are not `0` or `false`